### PR TITLE
update faq for os/hardware

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @eseidelGoogle

File: faq.md [hunk 1]
```diff
@@ -76,7 +76,14 @@ Sky was the codename of an earlier version of Flutter.
 
 ## What devices and OS versions does Flutter run on?
 
-Flutter apps run on Android Jelly Bean, v16, 4.1.x or newer, and iOS 8 or newer.
+Mobile operating systems: Android Jelly Bean, v16, 4.1.x or newer, and
+iOS 8 or newer.
+
+Mobile hardware: 64-bit iOS devices (for example, iPhone 5S and newer),
+and 32-bit ARM Android devices.
```
Comment (eseidelGoogle) [inline faq.md]: Just remove the "32-bit" from this sentence and it's correct.

File: faq.md [hunk 2]
```diff
@@ -76,7 +76,14 @@ Sky was the codename of an earlier version of Flutter.
...
+
+We support developing Flutter apps with Android emulator and iOS simulator.
```
Comment (eseidelGoogle) [inline faq.md]: I would say "emulators" since we support genymotion, etc.

**Comment by eseidelGoogle** [comment]
lgtm with edits.

File: faq.md (hunk 1 above)
Comment (sethladd) [inline faq.md]: thanks. done.

File: faq.md (hunk 2 above)
Comment (sethladd) [inline faq.md]: changed.

**Pull request merged**